src/Core/Mesh/Wrapper/TopologicalMeshConvert.cpp

```diff
@@ -88,35 +89,50 @@ void MeshConverter::convert( const TriangleMesh& in, TopologicalMesh& out ) {
     // Delete old data in out mesh
     out = TopologicalMesh();
     out.garbage_collection();
+    out.request_halfedge_normals();
     out.request_vertex_normals();
-    using vMap = std::unordered_map<Vector3, TopologicalMesh::VertexHandle, hash_vec>;
+    typedef std::unordered_map<Vector3, TopologicalMesh::VertexHandle, hash_vec> vMap;
```

Comment:
Why switching back from `using` to `typedef` ?

Reply:
well, I forgot this one during merge/patch